tests/pytorch/test_pytorch_model_export.py

```diff
@@ -813,3 +813,74 @@ def test_sagemaker_docker_model_scoring_with_sequential_model_and_default_conda_
     np.testing.assert_array_almost_equal(
         deployed_model_preds.values[:, 0], sequential_predicted, decimal=4
     )
+
+
+@pytest.fixture
+def create_artifact(tmpdir):
+    artifact_file_name = "requirements.txt"
+    fp = tmpdir.mkdir("artifacts").join(artifact_file_name)
```

Comment:
```suggestion
    fp = tmpdir.join(artifact_file_name)
```

Do we need `mkdir("artifacts")` here?

Reply:
Done